docs/src/manual/integration.md

```diff
@@ -15,3 +15,27 @@ Now, to compute the right side, i.e the integral on the reference shape, quadrat
 ```
 
 A specific procedure is applied to compute integrals on a face of a cell (i.e a surfacic integral on a face of a volumic element).
+
+## Defining a (bi)linear form
```

Comment:
This part could be in the paragraph `Assemble-(bi)linear-forms`

Reply:
Sure, but here it's just to introduce the concept of "Measure", I will write a similar text in the `assemble` page